### ISSUE TITLE
Improve accessibility of map search bar

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1193]
**Please complete the following information** 
Site repository URL: All sites
Open SDG version: All

**Describe the bug**
When users type a search, a list of suggested searches appears. Users can navigate this with the cursor keys; however, screen reader users are not made aware that this list has appeared. Voice activation users found that text dictated into the Search field, the autocorrect options were not triggered, so they were unable to use this control. 

**To Reproduce**
With screen reader on
1. Go to https://sdgdata.gov.uk/3-a-1/
2. Click on the Map tab
3. Click on the Map search icon to activate map
4. Enter 'north east'
5. See error

**Expected behavior**
Screen reader users are able to use Map search result controls

**Additional context**
From accessibility report:
WAI-ARIA roles can be utilised to ensure that all behaviour, information and functionality is exposed to assistive technologies. For more information, please refer to [WAI-ARIA Authoring Practices 1.1: Combobox with Listbox Popup Examples](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html).

Please note that Dragon dictation does not trigger certain keyboard and mouse events. If the field relies on JavaScript event handlers to see if its populated, these may not be triggered by dictation.

**Comment by LucyGwilliamAdmin** [comment]
Closing this as went into 1.4.0
